src/menu/star_select.c

```diff
@@ -378,6 +378,7 @@ Gfx *geo_act_selector_strings(s16 callContext, UNUSED struct GraphNode *node) {
 #endif
     if (callContext == GEO_CONTEXT_RENDER) {
 #ifdef ENABLE_N3DS_3D_MODE
+        gDPSetIod(gDisplayListHead++, iodStarSelect);
```

Comment:
did you try adding these AFTER the flush?

The flush results in a call to draw_triangles